### PR TITLE
React / Redux capstone

### PR DESCRIPTION
**Pull request by ReshyResh**
# Project requirements 
    - Use React documentation.
    - Use React components.
    - Use React props.
    - Use React Router.
    - Connect React and Redux.
    - Handle events in a React app.
    - Write integration tests with a React testing library.
    - Use styles in a React app.
    - Use React life cycle methods.
    - Apply React best practices and language style guides in code.
    - Use store, actions and reducers in React.

# Features

    - The app is fetching data from the API only once, on render.
    - While the data is loading the user is presented with a loading screen.
    - The user can sort the data by clicking the buttons on the home page.
    - When a card is clicker, the user is presented a page with details about the selected coin.
    - The user can scroll between the coins by using "Next" and "Prev" buttons at the bottom of the page.
    - When graph is clicked, the app fetches data about the selected coin to display a graph of the price.

# Screenshot

Senzanome

# Live demo
### Click [here](https://reshyresh.github.io/Capstone-React-Redux/)

# Video presentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+### Click [here](https://reshyresh.github.io/Capstone-React-Redux/)
+
+# Installation 
+
+To run the website locally:
+  - Clone the repo by running `git clone https://github.com/ReshyResh/Capstone-React-Redux.git`.
+  - Jump to that directory by using `cd Capstone-React-Redux ` .
+  - Run `npm install` to install dependencies.
+  - Run `npm run start` to launch the live server.
+
```
Comment (arslanbisharat) [inline README.md:33]: - [ ]  Let's make sure you have added instructions on how to run tests for this project. You can add something like this: 
` - Run npm run test to run the tests.`

File: src/__test__/CoinCard.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (arslanbisharat) [inline src/__test__/CoinCard.test.js:1]: - [ ] Let's make sure we have all the tests passing. 
Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Your pull request should be from Feature to Development but it is from Development to Main in your case. Please make the changes and request another review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
#### Hello @ReshyResh, 
Your project is complete!  All your tests are passing and you have met all requirements for this project. It is a thumbs up for me. 👍🏽  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**